Improve README by updating notes about how to use Studio (#688)
elan-ev/opencast-studio: README.md

## Usage

There are mainly three ways how to use Opencast Studio.

### Integrated in Opencast

Starting with Opencast 8.2, Opencast Studio is integrated into and shipped with

> Opencast itself. If you have an Opencast system already, this will be the

Reviewer: ```suggestion
Opencast itself. If you have an Opencast system already, this is the
```

}
```

### Self-hosted standalone version

As Studio is a client-only application, you can simply build it and then serve

> the resulting static files. This is rarely necessary, however.

Reviewer: ```suggestion
the resulting static files.
```
